File: iosApp/iosApp/Models/UserProfile.swift
import Foundation

struct UserProfile {
    let name: String
    let email: String
    let phone: String
    let address: String
    let location: String
    let initials: String
    let preferences: [UserPreference]
}

struct UserPreference: Identifiable {
    let id: String
    let key: String
    let label: String
    var enabled: Bool
}

File: androidApp/src/main/kotlin/com/example/app/android/model/UserProfile.kt
package com.example.app.android.model

data class UserProfile(
    val name: String,
    val email: String,
    val phone: String,
    val address: String,
    val location: String,
    val initials: String,
    val preferences: List<UserPreference>
)

data class UserPreference(
    val key: String,
    val label: String,
    val enabled: Boolean
)
